docs/presto-admin-commands.rst

```diff
@@ -374,11 +374,31 @@ defaults to /tmp.
 Example
 -------
 
-.. code-block:: none
+.. code-block:: text
 
     ./presto-admin file copy etc/presto/kafka-tabledef.json /etc/presto
 
 
+********
+file run
+********
+
+.. code-block:: text
+
+    presto-admin script run <local-path-to-script> [<remote-dir-to-put-script>]
+
+Use this command to run an arbitrary script on a cluster. It copies the script
+from its local location to the specified remote directory (defaults to /tmp),
+makes the file executable, and runs it.
+
+Example
+-------
+
+.. code-block:: text
+
+    ./presto-admin script run /my/local/script.sh
```

Comment:
file run here

Reply:
OK, fixed with **git commit --amend** so that it still looks like one commit.